src/util/symbol_table_base.h

```diff
@@ -92,10 +92,12 @@ class symbol_table_baset
   /// Find a symbol in the symbol table for read-only access.
   /// \param name: The name of the symbol to look for
   /// \return A reference to the symbol
-  /// \throw `std::out_of_range` if no such symbol exists
   const symbolt &lookup_ref(const irep_idt &name) const
   {
-    return symbols.at(name);
+    const auto symbol = lookup(name);
+    INVARIANT(
+      symbol, "`" + id2string(name) + "` must exist in the symbol table.");
```

Comment:
Maybe use `'` rather than ` to finish your quote.

Reply:
Done.